tools/querytee/proxy_endpoint.go

```diff
@@ -174,6 +199,8 @@ func (p *ProxyEndpoint) executeBackendRequests(req *http.Request, resCh chan *ba
 				"query", query,
 				"user", req.Header.Get("X-Scope-OrgID"),
 				"err", err,
+				"expected-response-duration", expectedResponse.elapsedTime,
+				"actual-response-duration", actualResponse.elapsedTime,
```

Comment:
[nit] "latency" might be clearer here (also applies below)

```suggestion
				"expected-response-latency", expectedResponse.elapsedTime,
				"actual-response-latency", actualResponse.elapsedTime,
```

Reply:
We currently emit `backend_request_duration_seconds` as a metric. I think keeping these as duration makes it more consistent

Reply:
I missed that, makes sense to keep it consistent.